ironfish/README.md

```diff
@@ -26,10 +26,10 @@ This represents the IronfishConfig and all of it's options. It's a hierarchical
 ```
 
 ### FileSystem
-This is an abtraction on top of any file system related APIs like `path` and `fs` in node. It makes it so you can perform file and file system related methods in a way that works in the browser and node. `NodeFileSystem` is one implementation that works for node.
+This is an abtraction on top of any file system-related APIs like `path` and `fs` in node. It makes it so you can perform file and file system-related methods in a way that works in the browser and node. `NodeFileSystem` is one implementation that works for node.
```

Comment:
Isn't the word here "file system" not "system-related"? What's the explanation here?